yb_migrate/cmd/exportSchema.go

```diff
@@ -61,25 +80,17 @@ func exportSchema() {
 
 	switch source.DBType {
 	case ORACLE:
-		utils.PrintIfTrue("preparing Ora2Pg for schema export from Oracle\n", source.VerboseMode, !source.GenerateReportMode)
```

Comment:
anyspecific reason to remove these lines? we can still have them with verbose mode

Reply:
These lines provide no valuable information to the user. And the terms `ora2pg` and `pg_dump` are internal implementation details.